docs/content/functions/sockaddr.md

```diff
@@ -372,4 +370,4 @@ $ gomplate -i '{{ sockaddr.GetInterfaceIPs "en0" }}'
 10.0.0.28 fe80::1f9a:5582:4b41:bd18
 ```
 
-[RFC 6890]: http://tools.ietf.org/html/rfc6890
+[1]: http://tools.ietf.org/html/rfc6890
```

Comment:
why this change?

Reply:
The `[RFC 6890][]` links are totally valid markdown - they all produce correct HTML:

```html
<a href="http://tools.ietf.org/html/rfc6890">RFC 6890</a>
```

There's an `[RFC 1918][]` link, on the other hand, which is invalid, because there's no associated `[RFC 1918]: http://tools.ietf.org/html/rfc1918` footer (which is a separate bug).

Since this PR is about removing `private`, please revert the `RFC 6890` changes.

Reply:
Ah, that's my bad. I meant the broken one. Apologies.